src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,33 @@
+package seedu.addressbook.data.person;
+
+public class Contact {
+
+    public String value;
+    protected boolean isPrivate;
+
+    public Contact() {
```

Comment:
The bot is suggesting you remove this constructor because it is empty. The correct implementation would have been for you to pass in the value, isPrivate, regex and the error message to the constructor and to use them appropriately. 

Empty constructors are really unnecessary.